### PR TITLE
Ability to skip impact analysis for a set of files

### PR DESCRIPTION
**Pull request by dennis-behm**
Use case is described in #59 .

### CODE REVIEW

File: utilities/ImpactUtilities.groovy [hunk 1]
```diff
@@ -386,6 +392,56 @@ def matches(String file, List<PathMatcher> pathMatchers) {
...
+	//	{
+	//		Properties impactProperties = new Properties()
+	//		propertyFile.withInputStream { impactProperties.load(it) }
+	//		impactChanges = impactProperties.get('impact_changes')
+	//		if (impactChanges != null){
+	//			if (impactChanges.toBoolean()==false) return false
+	//		}
+	//	}
+	//	return true // default
+}
```
Comment (jbyibm) [inline utilities/ImpactUtilities.groovy:425]: Can you remove this code in comments ?

File: utilities/ImpactUtilities.groovy [hunk 2]
```diff
@@ -386,6 +392,56 @@ def matches(String file, List<PathMatcher> pathMatchers) {
...
+	if (property) {
+		property.split(',').each{ filePattern ->
+			if (!filePattern.startsWith('glob:') || !filePattern.startsWith('regex:'))
+				filePattern = "glob:$filePattern"
+			PathMatcher matcher = FileSystems.getDefault().getPathMatcher(filePattern)
+			pathMatchers.add(matcher)
+		}
+	}
+	return pathMatchers
+}
```
Comment (jbyibm) [inline utilities/ImpactUtilities.groovy:443]: I think that this method can be merged with createExcludePatterns the 2 are very similar. Maybe createPathMatcherPattern can be use in both cases with the property parameter.

**Review by jbyibm — CHANGES_REQUESTED** [review]
See comments.

**Review by jbyibm — APPROVED** [review]
Look good

File: utilities/ImpactUtilities.groovy (hunk 2 above)
Comment (dennis-behm) [reply utilities/ImpactUtilities.groovy:412]: Makes sense, I have updated that.

File: utilities/ImpactUtilities.groovy (hunk 1 above)
Comment (dennis-behm) [reply utilities/ImpactUtilities.groovy]: The initial idea was to have a choice, but you are right. We should move this sample snippet to https://github.com/ibm/dbb

**Comment by dennis-behm** [comment]
### Sample usage

Specify the files, for which you would skip the impact analysis at 
https://github.com/IBM/dbb-zappbuild/blob/b55da1e5ea756e805f4581048f5105942c1c2708/samples/MortgageApplication/application-conf/application.properties#L27-L31

**Sample configuration** to skip analysis for the impacts for epsmtout.cpy. Please be aware, that this uses the glob syntax to identify them:
```
skipImpactCalculationList=**/epsmtout.cpy
```

During an **impactBuild** the impact analysis is skipped, which is reported in the build log in verbose mode:
```
..
** Impact analysis for MortgageApplication/copybook/epsmtout.cpy has been skipped due to configuration.
..
```

**Pull request merged**